core/binding/irb_spec.rb

```diff
@@ -1,11 +1,19 @@
 require_relative '../../spec_helper'
 
 describe "Binding#irb" do
+  before :each do
+    @env_home = ENV["HOME"]
+    ENV["HOME"] = CODE_LOADING_DIR
+  end
+
+  after :each do
+    ENV["HOME"] = @env_home
+  end
+
   it "creates an IRB session with the binding in scope" do
     irb_fixture = fixture __FILE__, "irb.rb"
-    irbrc_fixture = fixture __FILE__, "irbrc"
 
-    out = IO.popen([{"IRBRC"=>irbrc_fixture}, *ruby_exe, irb_fixture], "r+") do |pipe|
+    out = IO.popen([*ruby_exe, irb_fixture], "r+") do |pipe|
```

Comment:
Rather how about to change states only in the `IO.popen`-ed process? 

```suggestion
    dir = CODE_LOADING_DIR
    envs = {"IRBRC" => nil, "XDG_CONFIG_HOME" => nil, "HOME" => dir}

    out = IO.popen([envs, *ruby_exe, irb_fixture, chdir: dir], "r+") do |pipe|
```